static/css/page-user.less

```diff
@@ -104,6 +104,16 @@ div#subjectsPage{
   }
 }
 
+div#lists{
+  columns: 1;
+  @media only screen and (min-width: @width-breakpoint-tablet) {
+    columns: 2;
+  }
+  @media only screen and (min-width: @width-breakpoint-desktop) {
```

Comment:
This appears to render the whole list in a single column  for me at desktop breakpoint.

<img width="548" alt="Screen Shot 2021-01-07 at 5 45 20 PM" src="https://user-images.githubusercontent.com/148752/103964369-25c62000-5110-11eb-9aaa-f819360ce383.png">